### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by arturo-seijas**
Add the workflows currently used by IS charms to be able to leverage them as starter workflows

### CODE REVIEW

**Comment by mthaddon** [comment]
I'm a little concerned that this is an organisation-wide change (i.e. will affect all future repos in the canonical org), but seems to be IS Charms-specific (e.g. canonical/is-charms as CODEOWNERS but also the workflows as well). Can you explain how other teams would be able to define and use other starter workflows, especially since all of the filenames here seem quite generic?

**Comment by arturo-seijas** [comment]
> I'm a little concerned that this is an organisation-wide change (i.e. will affect all future repos in the canonical org), but seems to be IS Charms-specific (e.g. canonical/is-charms as CODEOWNERS but also the workflows as well). Can you explain how other teams would be able to define and use other starter workflows, especially since all of the filenames here seem quite generic?

Yes, I still plan to add a README here and in the referenced workflows before merging. Thanks for the reminder

File: README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Starter workflows
+
+## Description
+
+This repository includes Canonical's starter workflows.
+
+## Usage
+
+To use the workflows in your repository, you can follow (the instructions provided by GitHub's documentacion)[https://docs.github.com/en/actions/using-workflows/using-starter-workflows] or just copy the files.
```
Comment (mthaddon) [inline README.md:9]: s/documentacion/documentation/

**Review by gtrkiller — DISMISSED** [review]
LGTM

**Review by mthaddon — DISMISSED** [review]
LGTM

**Review by merkata — DISMISSED** [review]
Nice

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+      editorial revisions, annotations, elaborations, or other modifications
+      represent, as a whole, an original work of authorship. For the purposes
+      of this License, Derivative Works shall not include works that remain
+      separable from, or merely link (or bind by name) to the interfaces of,
+      the Work and Derivative Works thereof.
+
+      "Contribution" shall mean any work of authorship, including
+      the original version of the Work and any modifications or additions
+      to that Work or Derivative Works thereof, that is intentionally
+      submitted to Licensor for inclusion in the Work by the copyright owner
```
Comment (gtrkiller) [inline LICENSE:52]: Just wondering, should it be "to the licensor" instead of "to licensor"?

Comment (arturo-seijas) [reply LICENSE:52]: That's how the license is redacted https://www.apache.org/licenses/LICENSE-2.0

**Review by weiiwang01 — APPROVED** [review]

**Pull request merged**